lib/setup.js

```diff
@@ -759,19 +757,16 @@ function processCommand(command, args, params, callback) {
                 name = name.split('@')[0];
             }
 
-            if (!name) {
-                console.log('Please provide the name of the adapter to rebuild');
-                return void callback(EXIT_CODES.INVALID_ADAPTER_ID);
-            }
-
-            const rebuildCommand = params.install ? 'install' : 'rebuild';
-            installNpm(name, rebuildCommand, (err, _adapter) => {
-                if (err) {
-                    processExit(err);
-                } else {
+            console.log(`Rebuilding ${name}... (system call)`);
+            tools.rebuildNodeModules({
```

Comment:
One below.. smartphone 😀

Reply:
Ah, because `processCommand` isn't `async` - I didn't touch that.

Reply:
Then let’s just add an async prefix to the function and use async/await -shouldn’t hurt.